lib/net.js

```diff
@@ -1310,8 +1307,10 @@ Server.prototype._listen2 = function(address, port, addressType, backlog, fd) {
     this.unref();
 
   process.nextTick(emitListeningNT, this);
-};
+}
 
+Server.prototype._setUpListenHandle = setUpListenHandle;
+Server.prototype._listen2 = setUpListenHandle;  // legacy alias
```

Comment:
Not sure if `_listen2` should be simply removed (would anyone rely on a method with a name like this?)...`_setUpListenHandle` can be put into a internal symbol, though.